### ISSUE TITLE
What is not that great about data.nasa.gov : start a thread below, what would you change?

### ISSUE THREAD

**Issue by JustinGOSSES** [nasa/data-nasa-gov-frontpage#13]

**Comment by niquet** [comment]
# First and foremost

I love the website's structure and how you navigate through it reading the content. All of my concerns regard minor changes. As we say in German, "Jammern auf hohem Niveau". Let's jump right in.

## Desktop Version

1. The Logo

As much as it seems to be catching attention, a logo with all caps can be harder to read. Just as it is seen in the "Open Innovation Sites" section, it's better to make it lowercase.

Before:
![Bildschirmfoto 2020-06-25 um 22 23 28](https://user-images.githubusercontent.com/65405206/85791971-fda63f00-b732-11ea-9937-c6cb6ecaa02c.png)

After:
![Bildschirmfoto 2020-06-25 um 22 23 56](https://user-images.githubusercontent.com/65405206/85792003-0bf45b00-b733-11ea-8248-664d5cdeb1f9.png)

2. The Boxes within the header

Align the height of the two boxes (it's not necessary for functionality, but it levels out the optic).
I'd change the first box to the light-theme as well. It's an important element and the dark letters on the white background catch the viewers attention. Also the user is able to process the text slightly faster in this setting.
For even better usability change the text within the button to all uppercase or just capitalized.

Before:
![Bildschirmfoto 2020-06-25 um 22 30 00](https://user-images.githubusercontent.com/65405206/85793519-6d1d2e00-b735-11ea-8aa4-ee6eb6857f90.png)

After:
![Bildschirmfoto 2020-06-25 um 22 43 16](https://user-images.githubusercontent.com/65405206/85793724-b66d7d80-b735-11ea-832f-0f5e9aee2131.png)

3. The headings of each section

The split between the headings and the content is a great idea and its visually pleasing. Just keep the "data.nasa.gov" consistent throughout the page.

4. Align the pages content centered

5. Give those buttons more space to get the most usability out of them

6. Rethink the text before and within the Submit an issue button

![Bildschirmfoto 2020-06-25 um 23 17 01](https://user-images.githubusercontent.com/65405206/85796335-0e0de800-b73a-11ea-980d-f537c391d748.png)

## Mobile Hints

1. The boxes in the header align well when you switch the device. They tend to do weird things if you just adjust the window size on a laptop or bigger screen.

2. The menu is nit user friendly in the mobile version.

3. The rest is just amazing.

## My final example

I know these are not necessary changes to make the website more functional and you guys at NASA are doing way more important things at the moment. I just enjoy what you do and how you give people the chance to learn and gain insight from your data. Keep up the amazing work, maybe someday my contributions will have a bigger impact.

This was my very first issue to work on, and I enjoyed thinking about some UX changes. Haven't looked at the functionalities yet and / or the source code in this repository, I'll give it a shot later.

Before:
![2020-06-25-22-18-nasa github io](https://user-images.githubusercontent.com/65405206/85796923-0bf85900-b73b-11ea-985c-ca16db4fba14.png)

After:
![2020-06-25-23-17-nasa github io](https://user-images.githubusercontent.com/65405206/85797269-980a8080-b73b-11ea-8e7a-46ad48692a2a.png)

**Comment by JustinGOSSES** [comment]
@niquet These is some great feedback ! Thank you

I slightly disagree about the theme color change in point #2. 
#4 I would have to see it in practice to have a good opinion. 

All the other items sound fine.

I won't have time to work on any of these for a week or more. 

Please feel free to submit a pull request. As you have several changes here and not all might be integrated it is probably good to start from the up-to-date parent repo, fork, create new branch, make isolated change , then submit pull request. After each pull request, make new changes in a different branch. 

This way we can integrate any changes that are easy to approve from those that require a bit of additional work.

**Comment by niquet** [comment]
Yes, that sounds great. I’ll get to it during the upcoming week. Had to deal with work the last week as well.

I’ll let you know when it’s done.
